### PR TITLE
13.0 mig account analytic parent

### PR DESCRIPTION
**Pull request by pegonzalezspesol**
Work in Progress..

### CODE REVIEW

**Comment by pegonzalezspesol** [comment]
Work done:
- do it installable on v13
- fix tests exceptions
@angelmoya, @aheficent please review

File: account_analytic_parent/models/account_analytic_account.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+    @api.constrains("active")
+    def check_parent_active(self):
+        for account in self:
+            if (
+                account.active
+                and account.parent_id
+                and account.parent_id not in self
+                and not account.parent_id.active
+            ):
```
Comment (angelmoya) [inline account_analytic_parent/models/account_analytic_account.py]: you can use if self.filtered( lamda a: a.active and ..... instead of for .. if

**Review by angelmoya — APPROVED** [review]
Small remark but LGTM

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM Functional + code

**Comment by wpichler** [comment]
You should also take into account this PR #287

**Comment by AaronHForgeFlow** [comment]
This need rebase

**Comment by AaronHForgeFlow** [comment]
@pegonzalezspesol  it seems .isort.cfg has been updated in base branch. Can you rebase so we can merge?

**Pull request merged**